### PR TITLE
Edited snapshot retention note

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by MarkWoulfeNeo4j** [comment]
[Fix backups retention info](https://trello.com/c/YtdfZT1V/265-fix-backups-retention-info)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-aura-284.surge.sh

File: modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc [hunk 1]
```diff
@@ -29,12 +29,7 @@ label:AuraDB-Enterprise[]
 
 A *Scheduled* snapshot is a snapshot that is automatically triggered when you first create an instance, when changes to the underlying system occur (for example, a new patch release), and at a cadence depending on your plan type.
 
-Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Enterprise instances.
-
-[NOTE]
-====
-For Enterprise instances running Neo4j version 4, only one scheduled snapshot will be retained per day after 7 days.
-====
+Scheduled snapshots are run automatically once a day for Professional instances and once an hour for Enterprise instances. Only one scheduled snapshot is retained per day after 7 days for Enterprise instances.
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc:32]: One sentence per line, but also curious whether we mention if we retain snapshots at all for Professional instances?

[the remark this answers is not in the document]
Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/auradb/managing-databases/backup-restore-export.adoc:32]: Professional scheduled snapshots are only kept in the system for 7 days anyway, which we mention further up the page. This is to point out that only one of the possible 24 daily snapshots will be retained after a certain period.

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**